### DISCUSSION TITLE
Emergency Access / Legacy Access

### DISCUSSION THREAD

**Discussion by hj-collab** [ente-io/ente#503] [Enhancements]
I want to give access to my Ente account to my family after my death. A feature like Bitwarden Emergency Access or Google Inactive Account Manager will be great to have.

**Comment by vishnukvmd** [comment]
Hey, we're already working on this for Photos. Since the root account is the same for both Photos and Auth, the feature will be accessible for both.

**Comment by gi-yt** [comment]
Is there a public repository for this, I would like to contribute to this feature

**Comment by ua741** [comment]
https://help.ente.io/photos/features/legacy/#legacy
